schema.sql

```diff
@@ -1,5 +1,9 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
-);
+id BIGSERIAL NOT NULL PRIMARY KEY,
+name VARCHAR(300) NOT NULL,
+date_of_birth DATE NOT NULL,
+escape_attempts INTEGER,
+neutered BOOLEAN,
+weight_kg DECIMAL );
```

Comment:
Please note that you are to add the screenshots of your query results to the pull request description, so as to see if the results are correct. This also helps to ensure that you run the queries with success 👍 

![image](https://github.com/mohashyne/vet_clinic/assets/20537712/fe63be96-9d2d-406a-96ab-667571b1b606)

Reply:
Of course I did add the links for running the queries, did you click on the link to see the screenshots at all ?